api/resources/translations/messages-ne.properties

```diff
@@ -765,7 +765,8 @@ messages.f.report_accepted =
 messages.f.report_accepted_parent = 
 messages.generic.no_provided_patient_id = 
 messages.generic.provided_patient_id_not_unique = तपाइँले पठाउनुभएको आईडी पहिल्यै प्रयोगमा छ।  
-messages.generic.provided_phone_not_valid = तपाइँले पठाउनुभएको फोन नम्बर मिलेन । फेरी प्रयास गर्नुहोस्।
+messages.generic.provided_phone_not_valid = तपाइँले पठाउनुभएको फोन नम्बर मिलेन। कृपया पुनः प्रयास गर्नुहोस्।
+messages.invalid_phone= तपाइँले पठाउनुभएको फोन नम्बर मिलेन। कृपया पुनः प्रयास गर्नुहोस्।
```

Comment:
I think you removed this key from the en translation file so it should also be removed here?

Reply:
@garethbowen Thanks Fixed.